container-startup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright (c) 2016 Intel Corporation 
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export SERVER_IP=$1
+export SERVER_PORT=$2
+export DB_SERVER_IP=$3
+export DB_PORT=$4
+export DB_NAME=$5
+export CPU_COUNT=$6
+
+DB_URL=mongodb://${DB_SERVER_IP}:$DB_PORT/$DB_NAME
```

Comment:
Please add a comment stating assumptions such as, here it's assumed that the mongodb server is already running at the server and port specified in the DB_URL.